scripts/zones/Bastok_Markets/npcs/Gulldago.lua

```diff
@@ -4,16 +4,116 @@
 -- Type: Tutorial NPC
 -- !pos -364.121 -11.034 -167.456 235
 -----------------------------------
+local ID = require("scripts/zones/Bastok_Markets/IDs")
+require("scripts/globals/status")
+require("scripts/globals/keyitems")
+require("scripts/globals/settings")
+require("scripts/globals/npc_util")
+-----------------------------------
+
 
-function onTrade(player,npc,trade)
-end;
+function onTrade(player, npc, trade)
+end
 
-function onTrigger(player,npc)
-    player:startEvent(535);
-end;
+function onTrigger(player, npc)
+    local tutorial = player:getCharVar("TutorialProgress")
+    if tutorial == 11 then
+        player:startEvent(535)
```

Comment:
Should this last even have some sort of ending var or complete progress so you dont get stuck in a loop here. Or is this last cs its default text when done forever?